### PR TITLE
Doc/529 cli flags

### PR DESCRIPTION
**Pull request by Moneyba**
## Description
Adding missing CLI flags to documentation

## Checks
- [ ] Project Builds
- [ ] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2568,6 +2587,10 @@ boost nuke -e <environment name>
 
 **Note**: This will delete everything in your stack, including databases. This action is **not** reversible!
 
+For a force delete without asking for confirmation, you can run `boost nuke -e <environment name> -f`.
```
Comment (rdiaz82) [inline docs/README.md:2590]: Are neccesary those double ` accents at the begining for highlight the code?

**Review by rdiaz82 — APPROVED** [review]
LGTM just a mimor doubt

File: docs/README.md [hunk 2]
```diff
@@ -2546,7 +2562,10 @@ boost deploy -e <environment name>
 The `<environment name>` parameter is the name of the [environment](#environments) you want to deploy.
 It will take a while, but you should have your project deployed to your cloud provider.
 
-If you make changes to your code, you can run `boost deploy -e <environment name>` again to update your project in the cloud.
+If you make changes to your code, you can run ``boost deploy -e` <environment name>` again to update your project in the cloud.
```
Comment (rdiaz82) [inline docs/README.md:2565]: are necessary these double accent?

File: docs/README.md [hunk 3]
```diff
@@ -391,6 +391,20 @@ parameters are as follows:
...
+
+| Flag                   | Short version | Description |
+| :--------------------- | :------------ | :---------- |
+| `--homepage`           | `-H`          | the website of this project |
+| `--author`             | `-a`          | author of this project |
+| `--description`        | `-d`          | a short description |
+| `--license`            | `-l`          | license used in this project |
+| `--providerPackageName`| `-p`          | package name implementing the cloud provider integration where the application will be deployed |
+| `--repository`         | `-r`          | the URL of the repository |
+| `--version`            | `-v`          | the initial version |
```
Comment (javiertoledo) [inline docs/README.md:404]: 💯 Great idea to use a table here!

**Review by javiertoledo — APPROVED** [review]
Great changes!

**Pull request merged**